Cargo.toml

```diff
@@ -27,8 +27,10 @@ tonic = { version = "0.8.0", optional = true }
 prost = { version = "0.11.0", optional = true }
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread"], optional = true }
 uuid = { version = "1.1.2", features = ["serde", "v4"], optional = true }
-bincode = { version = "1.3.3", optional = true }
+bincode = { version = "1.3.3" }
 futures-util = "0.3.21"
+sev = { version = "1.0.1", optional = true }
```

Comment:
It would be a really good idea to bump this to:

```toml
sev = { version = "1.1.0", optional = true }
```

As the extended report patches and the asked changes for public visibility were applied to that release.